recursio-client/Characters/Player.gd

```diff
@@ -76,6 +77,12 @@ func apply_input(movement_vector: Vector3, rotation_vector: Vector3, buttons: in
 		if action.ammunition > 0:
 			_aim_visuals.visible = true
 			_aim_visuals.get_child(timeline_index % 2).visible = aim_mode
+		else:
+			# QUESTION: should this be in ClientActionManager?
```

Comment:
No I think this is fine :+1:

Reply:
Also think this is fine, because the Player script is client side only as well.